vsphere.py

```diff
@@ -118,7 +118,7 @@ def _run(config_list):
                     env.read_metric_values()
                     logger.info("Sent metrics for env : {0}".format(env.get_instance_id()))
                 except Exception as e:
-                    logger.error("Failed to send metrics for env {0} : {1}".format(env.get_instance_id(), e))
+                    logger.exception("Failed to send metrics for env {0} : {1}".format(env.get_instance_id(), e))
```

Comment:
If you use logger.exception I don't think you need to put e in the format string since it should put it in the log message for you.